### PR TITLE
Greek translation for 2.0

### PR DESCRIPTION
**Pull request by tassos**
I am providing the Greek translation, solving issue #35

### CODE REVIEW

File: lang/el.po [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+msgid "HELP_OPTION_HELP\n"
+msgstr "    Εκτύπωσε αυτή τη βοήθεια και τερμάτησε.\n"
+
+msgid "HELP_OPTION_IMAGE\n"
+msgstr "    Όπως το --ascii, αλλά εκτυπώνει μία εικόνα χρησιμοποιώντας χαρακτήρες Unicode\n"
+       "    και χαρακτήρες διαφυγής τερματικού 256-χρωμάτων.\n"
+
+msgid "HELP_OPTION_LICENCELIST\n"
+msgstr "    Προσδιορίζει την λίστα με τις καλές άδειες ή οποία θα χρησιμοποιηθεί. Το FILE μπορεί να είνα και διαδρομή\n"
```
Comment (suve) [inline lang/el.po]: The `HELP_...` messages shouldn't be longer than 80 characters per line (as in, 80 characters between opening quote (`"`) and the closing newline+quote (`\n"`). You can break the message into multiple lines or move words and sentences between lines as you wish, as long as the meaning is kept.

File: lang/el.po [hunk 2]
```diff
@@ -0,0 +1,100 @@
+msgid ""
+msgstr ""
+"Language: en\n"
```
[the remark this answers is not in the document]
Comment (tassos) [reply lang/el.po]: Fixed!

File: lang/el.po [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Tassos Natsakis\n"
```
[the remark this answers is not in the document]
Comment (tassos) [reply lang/el.po]: Fixed as well!

File: man/el.man [hunk 4]
```diff
@@ -0,0 +1,96 @@
+.TH VRMS-RPM 1 "2018-05-18"
```
[the remark this answers is not in the document]
Comment (tassos) [reply man/el.man]: Updated the date

File: man/el.man [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+.SH NAME
+vrms-rpm - αναφορά μη ελεύθερων εγκατεστημένων προγραμάτων
+
+.SH SYNOPSIS
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
+
+.SH DESCRIPTION
```
[the remark this answers is not in the document]
Comment (tassos) [reply man/el.man]: Done!

File: lang/el.po (hunk 1 above)
Comment (tassos) [reply lang/el.po]: I think I fixed this too.

**Comment by tassos** [comment]
Thanks for the comments @suve. I think I fixed all the issues you found. Please let me know if you need further improvements!

**Pull request merged**